base/applications/sndvol32/sndvol32.c

```diff
@@ -1378,6 +1389,11 @@ CreateApplicationWindow(
                  MixerWindow);
     }
 
+    if (!RegisterHotKey(hWnd, HOTKEY_CTRL_S, MOD_CONTROL, 'S'))
+    {
+        return NULL;
```

Comment:
I would object here that failure to register a convenience hotkey is _**not**_ a reason to fail `CreateApplicationWindow()`.
Since you handle the hotkey deregistration in the `MainWindowProc()` function above, wouldn't it be more logical to also handle its registration in that same window proc, when handling the `WM_CREATE` message?
As an alternative you could do the registration and deregistration in the `_tWinMain()` function: the registration [before the `while (GetMessage())` loop](https://git.reactos.org/?p=reactos.git;a=blob;f=base/applications/sndvol32/sndvol32.c;hb=750bf12256323202fbe09209903c12a28e41f5b2#l1499), and the deregistration just before the [`DestroyWindow()` call](https://git.reactos.org/?p=reactos.git;a=blob;f=base/applications/sndvol32/sndvol32.c;hb=750bf12256323202fbe09209903c12a28e41f5b2#l1510).

Reply:
Fair point, I'll just move the registration to ```MainWindowProc()``` too and make it so it doesn't fail anything if the Hotkey isn't registered properly.